osu.Game/Database/DatabaseWorkingBeatmap.cs

```diff
@@ -14,8 +14,8 @@ internal class DatabaseWorkingBeatmap : WorkingBeatmap
     {
         private readonly BeatmapDatabase database;
 
-        public DatabaseWorkingBeatmap(BeatmapDatabase database, BeatmapInfo beatmapInfo, BeatmapSetInfo beatmapSetInfo, bool withStoryboard = false)
-            : base(beatmapInfo, beatmapSetInfo, withStoryboard)
+        public DatabaseWorkingBeatmap(BeatmapDatabase database, BeatmapInfo beatmapInfo, BeatmapSetInfo beatmapSetInfo = null, BeatmapMetadata metadata = null, bool withStoryboard = false)
+            : base(beatmapInfo, beatmapSetInfo, metadata, withStoryboard)

```

Comment:
These was supposed for a placeholder which has metadata available, but no actual beatmap.
Not sure if the placeholder is a good idea. Maybe I should revert these.